### PR TITLE
Use the separate config stack di-btm-cfg-$ENV.

### PR DESCRIPTION
**Pull request by phinterseer**
Removed parts that went into the config stack. Also some test refactoring.

Once this is merged:

I recommend deleting your ephemeral stacks before applying this change or there will be a lost config bucket flying around that you need to delete manually (which is also the alternative if you don't heed this advice :-) ).

### CODE REVIEW

**Comment by NKannan14** [comment]
@phinterseer Integration test changes looks good to me

File: integration_tests/helpers/envHelper.ts [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
   }`;
-}
 
-export { resourcePrefix };
+const configStackName = (): string =>
+  `di-btm-config-${
+    process.env.CONFIG_NAME !== undefined ? process.env.CONFIG_NAME : "dev"
+  }`;
+
+export { resourcePrefix, configStackName };
```
Comment (Si3rr4wow) [inline integration_tests/helpers/envHelper.ts:11]: nit pick but why not just `export` each `const`?

Comment (phinterseer) [reply integration_tests/helpers/envHelper.ts:11]: I'd prefer that as well, but that's how it's done across the codebase, so I didn't mess with it.

**Review by Si3rr4wow — APPROVED** [review]
LGTM

File: integration_tests/helpers/envHelper.ts (hunk 1 above)
Comment (Si3rr4wow) [reply integration_tests/helpers/envHelper.ts:11]: okily-dokily

**Review by markschnitziusgovuk — APPROVED** [review]
Yep, all looks good.

**Review by Si3rr4wow — APPROVED** [review]
👍

**Pull request merged**